plugins/coral-plugin-love/server/typeDefs.graphql

```diff
@@ -0,0 +1,70 @@
+enum ACTION_TYPE {
+
+    # Represents a Love.
+    LOVE
```

Comment:
@jde Every name we introduce here in the schema is global. 2 plugins that happen to use the same name anywhere will not work with each other. I leave it up to you, if we need a strategy now or later.